scanpipe/templates/scanpipe/resource_detail.html

```diff
@@ -54,76 +58,104 @@
   {{ detected_values|json_script:"detected_values" }}
   <script>
     let editor = ace.edit("editor", {
-      mode: "ace/mode/text",
-      autoScrollEditorIntoView: true,
-      wrap: true,
-      readOnly: true,
-      showPrintMargin: false,
-      highlightActiveLine: false,
-      highlightGutterLine: false,
-      fontSize: 15,
-      foldStyle: "manual",
-      fontFamily: "SFMono-Regular,Consolas,Liberation Mono,Menlo,monospace",
+        mode: "ace/mode/text",
+        autoScrollEditorIntoView: true,
+        wrap: true,
+        readOnly: true,
+        showPrintMargin: false,
+        highlightActiveLine: false,
+        highlightGutterLine: false,
+        fontSize: 15,
+        foldStyle: "manual",
+        fontFamily: "SFMono-Regular,Consolas,Liberation Mono,Menlo,monospace",
```

Comment:
Why?

Reply:
Do not change existing style conventions but adapt to it.

Reply:
indentation using prettifier

Reply:
Please revert and adapt to the existing style.
You're making the code review process more painful with those unwanted changes.

Reply:
tabs should be of 2 spaces or 4?

Reply:
I have been using tab of 4 spaces, just switched to 2.